cmd/gitserver/server/patch.go

```diff
@@ -105,17 +104,28 @@ func (s *Server) createCommitFromPatch(ctx context.Context, req protocol.CreateC
 		out, err := runWith(ctx, cmd, true, nil)
 		if err != nil {
 			resp.SetError(repo, argsToString(cmd.Args), string(out), errors.Wrap(err, "gitserver: "+reason))
-			log15.Info("command failed", "prefix", prefix, "command", argsToString(cmd.Args), "duration", time.Since(t), "error", err, "output", string(out))
+			s.Log.Info("command failed",
```

Comment:
You could not infer this due to a lack of context, but as we're preparing to plug in error reporting on top logging, we made the decision that an error being reported at the info level will be ignored. 

So here, I'm making the decision that this one will be reported, your code is totally correct. 

```suggestion
			s.Log.Warn("command failed",
```